Project.toml

```
@@ -27,7 +27,7 @@ Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
 
 [compat]
 AbstractTrees = "0.3"
-Adapt = "2.0, 3.0"
+Adapt = "2.0, 3.1"
```

```suggestion
Adapt = "2.0, 3"
```

For the Pkg resolver error.